src/client/sagas/fetch/index.js

```diff
@@ -13,7 +14,7 @@ export function* fetchSaga(
   try {
     yield put(actions.clearError(actionName))
     yield put(actions.sendingRequest(actionName, true))
-    return yield call(get, graphQuery, variables)
```

Comment:
We are doing posts everywhere. GraphQL supports both for queries and mutations. And doesn't demand on using a particular one. The biggest advantage of using posts that we can use caching on a database/intermediary level.